docs/changelog/93396.yaml

```diff
@@ -0,0 +1,51 @@
+pr: 93396
+summary: Add support for Reciprocal Rank Fusion to the search API
+area: Ranking
+type: feature
+issues: []
+highlight:
+  title: Add support for Reciprocal Rank Fusion to the search API
+  body: |-
+    This change at a high level adds global re-ranking on the coordinating
+    node at the end of query reduction prior to the fetch phase. The
+    singular re-ranking model added in this change is reciprocal rank fusion
+    (RRF) which follows the basic formula for merging `1...n` sets of
+    results sets together with `sum(1/(k+d))` where k is a ranking constant
+    and d is a document's scored position within a result set from a query.
+
+    The API for this change adds a `rerank` top-level element to the search
```

Comment:
the section is called rank and not rerank?

Reply:
Same comment as above.

Reply:
Fixed.